testsuite/MDAnalysisTests/analysis/test_gnm.py

```diff
@@ -80,6 +81,18 @@ def test_generate_kirchoff(universe):
        0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0])
 
 
+def test_gnm_SVD_fail(universe):
+    gnm = mda.analysis.gnm.GNMAnalysis(universe)
+    # Initilize attributes like self._ts 
+    # necessary for print the warning
+    gnm.run(stop=1)
+
+    with patch.object(np.linalg, "svd") as np_load_mock:
+        np_load_mock.side_effect = np.linalg.LinAlgError
+        with pytest.warns(UserWarning):
```

Comment:
I'd add a match on the warning here (MDA throws a ton of warnings all the time).

Reply:
True :)